package.json

```diff
@@ -4,10 +4,18 @@
   "dependencies": {
```

Comment:
- [ ] Inside your package.json, you should be having `react-dom` and `babel-loader`. These are required to successfully run the project.

Reply:
@Lucas-Masaba it was added using yarn I forgot to mention that my bad, anyway I added it using npm again.

Reply:
Alright @ouasamine